pyproject.toml

```diff
@@ -137,3 +162,31 @@ known-first-party = ["tardis", "carsus", "stardis"]
 
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
+
+[tool.setuptools]
+packages = ["tardis"]
+license-files = ["LICENSE.rst", "licenses/*.rst"]
+
+[tool.setuptools.package-data]
+tardis = ["data/*"]
+tardis_grid = ["tests/data/*", "./"]
+tardis_io = [
+    "configuration/schemas/*",
+    "configuration/tests/data/*", 
+    "model/readers/tests/data/*"
+]
+tardis_model_tests = ["data/*"]
+tardis_montecarlo_tests = ["data/*"]
+tardis_plasma_tests = ["data/*"]
+tardis_scripts = ["debug/*"]
+tardis_tests_integration_tests = ["**/*"]
+tardis_visualization = ["tools/tests/data/*"]
+
+[tool.pytest.ini_options]
+testpaths = ["tardis"]
+# astropy_header = true
```

Comment:
Why is this commented out?

Reply:
The tests were failing after I added those two lines so I was testing which is exactly causing the tests to fail.